pom.xml

```diff
@@ -88,6 +168,55 @@
     </dependencyManagement>
 
     <build>
+        <pluginManagement>
+            <plugins>
```

Comment:
Of course there's not much to comment on the API, I do feel about some of this configuration is bit of copy-paste cargo cult. But I might be wrong. However putting in a comment to explain why we're doing that would be helpful.

Reply:
It was indeed copied from Notifiers.
From memory I _think_ it is required, but I don't remember why.
I'll do some poking

Reply:
All but one seem required, so I've removed the dependency stuff for now (since I'm not sure why that's doing what it's doing really), and added a comment next to each plugin